pydantic/fields.py

```diff
@@ -1027,6 +1028,7 @@ class ComputedFieldInfo:
     examples: list[Any] | None
     json_schema_extra: JsonDict | typing.Callable[[JsonDict], None] | None
     repr: bool
+    dependencies: list[str]
```

Comment:
Why not `Sequence[str]` here? I suppose we're probably about to change this for nested values anyways...

Reply:
yeah, will change soon anyway, as per your suggestion.  But here, i had used `Sequence[str]` for what we accept from the user in the `computed_field` decorator... but cast that all to `list[str]` for uniformity on this internal object ("Postel's law" style)